ssh/ssh_tunnel.py

```diff
@@ -73,7 +73,7 @@ def remote_cmd(self, cmd, timeout=None, stdout=None):
                 return check_output(run_cmd, timeout=timeout)
         except TimeoutExpired as e:
             logging.exception('{} timed out after {} seconds'.format(cmd, timeout))
-            logging.debug('Timed out process output:\n' + e.output)
+            logging.debug('Timed out process output:\n' + e.output.decode())
```

Comment:
You could also do  `logging.debug('Timed out process output: {!r}'.format(e.output))`  - General useful strategy in logging.